GeosCore/sulfate_mod.F90

```diff
@@ -1000,7 +1000,7 @@ SUBROUTINE SRCSF30( Input_Opt, State_Grid, State_Met, State_Chm, TC2, RC )
 
     INTEGER                :: N_TRACERS
     INTEGER                :: IBINS
-    LOGICAL                :: ERRORSWITCH, SGCOAG = .TRUE.
+    LOGICAL                :: ERRORSWITCH, SGCOAG = .FALSE. ! bc,jrp - turn off subgridcoag 18/12/23
```

Comment:
I would recommend making this a field in `Input_Opt` so you don't have to define it in multiple locations (e.g. here and in carbon_mod.F90).

Reply:
Hi @msulprizio - currently we do not want users to have the option of turning SGCOAG to true because there is a resolution dependency in the related code that could lead to erroneous results at some resolutions. Would it be ok to leave these resolution-dependent code sections commented out (as a reference while we work on developments based on them) in the case that we might develop SGCOAG to work again in the future? Thanks for this suggestion! Will plan to use that setup if proper functionality of SGCOAG is restored in the future.

Reply:
Sounds good. We will keep it as is then.